mongorestore/mongorestore_test.go

```diff
@@ -100,7 +100,7 @@ func TestDeprecatedDBAndCollectionOptions(t *testing.T) {
 			So(buffer.String(), ShouldBeEmpty)
 		})
 
-		Convey("and a warning is issued in the deprecated case", func() {
+		Convey("and a warning is issued in the deprecated case", t, func() {
```

Comment:
These typos are from [TOOLS-2363](https://github.com/mongodb/mongo-tools/pull/256).

Reply:
You might need to fix the top level Convey as well. The test is currently failing 
 [2020/06/24 16:49:57.224] panic: Top-level calls to Convey(...) need a reference to the *testing.T.
 [2020/06/24 16:49:57.224] 		Hint: Convey("description here", t, func() { /* notice that the second argument was the *testing.T (t)! */ })  [recovered]
 [2020/06/24 16:49:57.224] 	panic: Top-level calls to Convey(...) need a reference to the *testing.T.
 [2020/06/24 16:49:57.224] 		Hint: Convey("description here", t, func() { /* notice that the second argument was the *testing.T (t)! */ })

Reply:
Turns out there are more errors with this test, even after fixing the typo. A new ticket will be created to address this.